### ISSUE TITLE
Add ARIN API

### ISSUE THREAD

**Issue by wallacemariadeandrade** [wallacemariadeandrade/BGPViewerTool#25]
API docs:

https://www.arin.net/resources/registry/whois/rws/api/

**Comment by wallacemariadeandrade** [comment]
**The default data format that's returned from API is XML, so it's necessary to specify header "Accept: application/json" to retrieve in JSON format.**

**Comment by wallacemariadeandrade** [comment]
Query to retrieve AS informations:

`http://whois.arin.net/rest/asn/53181`

Output:

```
{
    "asn": {
        "@xmlns": {
            "ns3": "http://www.arin.net/whoisrws/netref/v2",
            "ns2": "http://www.arin.net/whoisrws/rdns/v1",
            "$": "http://www.arin.net/whoisrws/core/v1"
        },
        "@copyrightNotice": "Copyright 1997-2021, American Registry for Internet Numbers, Ltd.",
        "@inaccuracyReportUrl": "https://www.arin.net/resources/registry/whois/inaccuracy_reporting/",
        "@termsOfUse": "https://www.arin.net/resources/registry/whois/tou/",
        "registrationDate": {
            "$": "2009-03-11T00:00:00-04:00"
        },
        "rdapRef": {
            "$": "https://rdap.arin.net/registry/autnum/52224"
        },
        "ref": {
            "$": "https://whois.arin.net/rest/asn/AS52224"
        },
        "endAsNumber": {
            "$": "53247"
        },
        "handle": {
            "$": "AS52224"
        },
        "name": {
            "$": "LACNIC-52224"
        },
        "resources": {
            "@copyrightNotice": "Copyright 1997-2021, American Registry for Internet Numbers, Ltd.",
            "@inaccuracyReportUrl": "https://www.arin.net/resources/registry/whois/inaccuracy_reporting/",
            "@termsOfUse": "https://www.arin.net/resources/registry/whois/tou/",
            "limitExceeded": {
                "@limit": "256",
                "$": "false"
            },
            "resource": [
                {
                    "@description": "Web Page",
                    "@media_type": "text/html",
                    "@rel": "SELF",
                    "@url": "http://lacnic.net/cgi-bin/lacnic/whois"
                },
                {
                    "@description": "Whois",
                    "@media_type": "application/x-whois",
                    "@rel": "SELF",
                    "@url": "whois.lacnic.net"
                }
            ]
        },
        "orgRef": {
            "@handle": "LACNIC",
            "@name": "Latin American and Caribbean IP address Regional Registry",
            "$": "https://whois.arin.net/rest/org/LACNIC"
        },
        "comment": {
            "line": [
                {
                    "@number": "0",
                    "$": "This AS is under LACNIC responsibility for further allocations to"
                },
                {
                    "@number": "1",
                    "$": "users in LACNIC region."
                },
                {
                    "@number": "2",
                    "$": "Please see http://www.lacnic.net/ for further details, or check the"
                },
                {
                    "@number": "3",
                    "$": "WHOIS server located at http://whois.lacnic.net"
                }
            ]
        },
        "startAsNumber": {
            "$": "52224"
        },
        "updateDate": {
            "$": "2009-03-17T15:29:07-04:00"
        }
    }
}
```

**Comment by wallacemariadeandrade** [comment]
Query to retrieve IP details:

`http://whois.arin.net/rest/ip/8.8.8.8`

Output: 

```
{
    "net": {
        "@xmlns": {
            "ns3": "http://www.arin.net/whoisrws/netref/v2",
            "ns2": "http://www.arin.net/whoisrws/rdns/v1",
            "$": "http://www.arin.net/whoisrws/core/v1"
        },
        "@copyrightNotice": "Copyright 1997-2021, American Registry for Internet Numbers, Ltd.",
        "@inaccuracyReportUrl": "https://www.arin.net/resources/registry/whois/inaccuracy_reporting/",
        "@termsOfUse": "https://www.arin.net/resources/registry/whois/tou/",
        "registrationDate": {
            "$": "2014-03-14T16:52:05-04:00"
        },
        "rdapRef": {
            "$": "https://rdap.arin.net/registry/ip/8.8.8.0"
        },
        "ref": {
            "$": "https://whois.arin.net/rest/net/NET-8-8-8-0-1"
        },
        "endAddress": {
            "$": "8.8.8.255"
        },
        "handle": {
            "$": "NET-8-8-8-0-1"
        },
        "name": {
            "$": "LVLT-GOGL-8-8-8"
        },
        "netBlocks": {
            "netBlock": {
                "cidrLength": {
                    "$": "24"
                },
                "endAddress": {
                    "$": "8.8.8.255"
                },
                "description": {
                    "$": "Reallocated"
                },
                "type": {
                    "$": "A"
                },
                "startAddress": {
                    "$": "8.8.8.0"
                }
            }
        },
        "resources": {
            "@copyrightNotice": "Copyright 1997-2021, American Registry for Internet Numbers, Ltd.",
            "@inaccuracyReportUrl": "https://www.arin.net/resources/registry/whois/inaccuracy_reporting/",
            "@termsOfUse": "https://www.arin.net/resources/registry/whois/tou/",
            "limitExceeded": {
                "@limit": "256",
                "$": "false"
            }
        },
        "orgRef": {
            "@handle": "GOGL",
            "@name": "Google LLC",
            "$": "https://whois.arin.net/rest/org/GOGL"
        },
        "parentNetRef": {
            "@handle": "NET-8-0-0-0-1",
            "@name": "LVLT-ORG-8-8",
            "$": "https://whois.arin.net/rest/net/NET-8-0-0-0-1"
        },
        "startAddress": {
            "$": "8.8.8.0"
        },
        "updateDate": {
            "$": "2014-03-14T16:52:05-04:00"
        },
        "version": {
            "$": "4"
        }
    }
}
```

**Comment by wallacemariadeandrade** [comment]
implemented in #37
